subsys/ipc/rpmsg_multi_instance/rpmsg_multi_instance.c

```diff
@@ -45,10 +43,11 @@ static uint32_t virtio_get_features(struct virtio_device *vdev)
 }
 
 #ifdef CONFIG_RPMSG_MULTI_INSTANCE_MASTER
-
 static void virtio_set_status(struct virtio_device *p_vdev, unsigned char status)
 {
-	struct rpmsg_mi_ctx *ctx  = metal_container_of(p_vdev, struct rpmsg_mi_ctx, vdev);
+	struct rpmsg_mi_ctx *ctx;
+
+	ctx = metal_container_of(p_vdev, struct rpmsg_mi_ctx, vdev);
```

Comment:
It's simply ugly and the line is way too long.

Reply:
This line is not too long. Zephyr's line length limit was extended to 100 characters. This is not mixing declaration and definition, but this is definition with initialization. "Ugly" is an opinion.

This change is based on your personal preference, as you pointed out. Next contributor may have different opinion on this and he will change this line as it was before. He can do that, if he feel that you code is ugly - this is based on your way thinking. Later, if you want to contribute to this file and you will see this line, you will change it, and so on...

I will accept this PR with this change, because it does not violate any of our standards and it is not functional change. I am just expressing that this is a bad practice IMO.

Personally, I think that your code looks better, but it's not the point.